### PR TITLE
hashi_vault lookup - docs: add commas, spacing into examples

### PR DESCRIPTION
**Pull request by rwaweber**
##### SUMMARY

- Adds commas into the examples
- Make spacing more consistent among examples

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

hashi_vault lookup plugin

##### ANSIBLE VERSION

```paste below
2.6.4
```

### CODE REVIEW

File: lib/ansible/plugins/lookup/hashi_vault.py [hunk 1]
```diff
@@ -56,27 +56,27 @@
 
 EXAMPLES = """
 - debug:
-    msg: "{{ lookup('hashi_vault', 'secret=secret/hello:value token=c975b780-d1be-8016-866b-01d0f9b688a5 url=http://myvault:8200')}}"
+    msg: "{{ lookup('hashi_vault', secret=secret/hello:value, token=c975b780-d1be-8016-866b-01d0f9b688a5, url=http://myvault:8200) }}"
```
Comment (felixfontein) [inline lib/ansible/plugins/lookup/hashi_vault.py]: I don't think this is valid Jinja2 syntax. Do you mean:
```suggestion
    msg: "{{ lookup('hashi_vault', secret='secret/hello:value', token='c975b780-d1be-8016-866b-01d0f9b688a5', url='http://myvault:8200') }}"
```
?

**Comment by acozine** [comment]
Thanks @rwaweber for working to make the docs better, and apologies for your PR not getting a response until today. 

On first glance, these changes look odd. Could you post output from the old examples and from your changed examples, so we can see both the problem and the solution?

**Comment by rwaweber** [comment]
Hey @acozine, no worries! Thanks for the feedback @felixfontein!

Unfortunately, I don't have output on hand -- but I distinctly remember the lookups not working in their documented examples, since kwargs are used in the input and currently its documented as a single string argument.

Example in the etcd lookup plugin using kwargs: https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/lookup/etcd.py#L64

Example single string parameter in this plugin: https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/lookup/hashi_vault.py#L59

**Comment by acozine** [comment]
Closing. I tested these changes using Ansible 2.7.4 against a dev HashiCorp vault server. The old examples successfully connect to the server. The changed examples suggested here return this error:

```
"An unhandled exception occurred while running the lookup plugin 'hashi_vault'. Error was a <type 'exceptions.IndexError'>, original message: list index out of range"
```

Although it's unusual for a modern Ansible lookup plugin, the `hashi_vault` plugin is expecting all its arguments in a single string. Here's a link to the [code that splits that string](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/lookup/hashi_vault.py#L260) to extract the args.

**Pull request closed without merging**